src/augtool.c

```diff
@@ -426,6 +427,28 @@ static void print_aug_error(void) {
     }
 }
 
+static void sigint_handler(int unused)
+{
+    (void)unused;
```

Comment:
Instead of keeping the compiler from complaining about the unused param this way, please annotate it with `ATTRIBUTE_UNUSED`, i.e. change this to

    static void sigint_handler(ATTRIBUTE_UNUSED int unused) {
      ...
    }